src/omv/imlib/framebuffer.h

```diff
@@ -34,6 +34,8 @@ typedef struct framebuffer {
     volatile int32_t tail;
     bool check_head;
     int32_t sampled_head;
+    // Disable autoflush policy on full to prevent stale frames.
+    bool disable_full_flush;
```

Comment:
```suggestion
```
This should be in sensor_t, it's not really a framebuffer thing.

Reply:
Or check for this flag in framebuffer_flush_buffers() ? but make sure it's initialized to zero.

Reply:
It gets zeroed in the main memset of the frame buffer header.